client/src/components/blog/blog.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './blog.css';
+
+class Blog extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      blog: {}
+    };
+  }
+
+  componentDidMount() {
+    fetch('/blogs/' + this.props.match.params.id)
+      .then(res => res.json())
+      .then(blog => {
+        this.setState({ blog });
+      });
+  }
+
+  delete(id) {
+    console.log(id);
+    fetch('/blogs/' + id, {
+      method: 'DELETE',
+      mode: 'CORS',
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }).then(res => {
+      this.props.history.push("/blogs");
+    }).catch(err => err);
+  }
+
+  render() {
+    return (
+      <div>
+        <h2> {this.state.blog.title}</h2>
+        <div className="textCenter">
+          <h3>Written by {this.state.blog.author} at {new Date(this.state.blog.date).toLocaleDateString('en-US')}</h3>
```

Comment:
Export to until's function, please
`new Date(this.state.blog.date).toLocaleDateString('en-US')`

Reply:
done